pkg/controller/eventing/inmemory/channel/reconcile_test.go

```diff
@@ -643,15 +643,15 @@ func makeVirtualService() *istiov1alpha3.VirtualService {
 		Spec: istiov1alpha3.VirtualServiceSpec{
 			Hosts: []string{
 				serviceAddress,
-				fmt.Sprintf("%s.%s.channels.cluster.local", cName, cNamespace),
```

Comment:
This could be a problem if existing configurations are using this host. I think this will also be a problem during reconciliation, because the VirtualService will be updated at a different time than whatever is watching the Channel's `status.address`.

Can we leave this hostname in for 0.4 so that users upgrading from 0.3 to 0.4 don't have address discontinuity when upgrading?